README.md

```diff
@@ -164,6 +164,7 @@ the previous snippet. A summary of all settings follows:
 | vastLoadTimeout        | number       | Override for default VAST load timeout in milliseconds for a single wrapper. The default timeout is 5000ms. |
 | vpaidAllowed           | boolean      | **DEPRECATED**, please use vpaidMode. |
 | vpaidMode              | VpaidMode(5) | VPAID Mode. Defaults to ENABLED. This setting,overrides vpaidAllowed. |
+| doNotPlayAdAfterContentStart | boolean | Prevent ads from starting after the content has started if an adtimeout occurred (preroll, midroll, postroll). The default value is false
```

Comment:
Looking to merge this change, but first could you change the variable name to imply the ad loading is slow. Ex. preventLateAdStart